libraries/fc/src/log/log_message.cpp

```diff
@@ -185,6 +185,13 @@ namespace fc
       my->args    = std::move(args);
    }
 
+   log_message::log_message( std::string format, variant_object args )
```

Comment:
unnecessary copy? or it can not be passed trough macros differently?

Reply:
this is ok as this class needs copy of this string anyway and constructor is using std::move inside